Nautono/RTOS_framework_v1/src/sail_wind.c

```diff
@@ -60,8 +66,59 @@ static enum status_code WIND_ParseMWV(void);
 static void WEATHERSTATION_InitPin(void);
 static void WS_TurnOn(void);
 static void WS_TurnOff(void);
+///// Needed? - KT ^^^ ///////
```

Comment:
We should keep this, in the previous sailboats we had the ability to toggle the power to all of the peripherals in an effort to conserve power. At the moment I do not know what pins will be connected to the toggles